src/Panel.js

```diff
@@ -61,20 +64,16 @@ const childContextTypes = {
 };
 
 class Panel extends React.Component {
-  constructor(...args) {
-    super(...args);
-    this.handleToggle = this.handleToggle.bind(this);
-  }
-
   getChildContext() {
     const { eventKey, id } = this.props;
-    let { getId } = this.context.$bs_panelGroup || {};
+    const idKey = eventKey == null ? id : eventKey;
 
     let ids;
-    let idKey = eventKey == null ? id : eventKey;
 
     if (idKey !== null) {
-      getId = getId || defaultGetId;
+      const panelGroup = this.context.$bs_panelGroup;
```

Comment:
If not in PanelGroup this causes failure on next line at `panelGroup.getId`.
So maybe use.
`const panelGroup = this.context.$bs_panelGroup | {};`